tutor/test_t1_58_ManageEcosystems.py

```diff
@@ -1648,21 +1687,68 @@ def test_content_analyst_verify_us_history_111270(self):
         """Verify question availability for College Biology.
 
         Steps:
+        Log in as Content Analyst
+        Open the drop-down menu
+        Click "QA Contents"
+        Select "US History" in the "available-books drop-down list"
+        Click a non-introductory section
 
         Expected Result:
+        Exercises are displayed.
         """
-        self.ps.test_updates['name'] = 't1.58.026' \
+        self.ps.test_updates['name'] = 't1.58.027' \
             + inspect.currentframe().f_code.co_name[4:]
         self.ps.test_updates['tags'] = [
             't1',
             't1.58',
-            't1.58.026',
-            '111270'
+            't1.58.027',
+            '111271'
```

Comment:
The test case ID tag number (line 1705) doesn't match the preview comment (line 1623), the function skip control (line 1685) or the function name (line 1686)